docs/dev-guide/integration/rest-api/repo/cud.rst

```diff
@@ -87,7 +87,8 @@ is centered around updating only that metadata.
 
 * :param:`?delta,object,object containing keys with values that should be updated on the repository`
 * :param:`?importer_config,object,object containing keys with values that should be updated on the repository's importer config`
-* :param:`?distributor_configs,object,object containing keys that are distributor ids, and values that are objects containing keys with values that should be updated on the specified distributor's config`
+* :param:`?distributor_configs,object,dict containing keys that are distributor ids, and values that are dictionaries containing plugin specific keys/value pairs`
```

Comment:
In JSON there is no such thing as a dict - it's called an object.

http://json.org/